os_windows_test.go

```diff
@@ -0,0 +1,82 @@
+// +build windows
+
+package ctxvfs
+
+import (
+	"testing"
+)
+
+func TestWindowsMount(t *testing.T) {
+
+	// keys are mount points,
+	// values contain expected results for osFS.resolve(key)
+	tests := map[string]map[string]string{
+		"": {
+			"C:/folder/file":  "C:\\folder\\file",
+			"D:/folder/file":  "D:\\folder\\file",
+			"/C:/folder/file": "C:\\folder\\file",
+			"/folder/file":    "",
+			"file":            "",
+			"/file":           "",
+		},
+		"C:": {
+			"C:/folder/file":  "C:\\folder\\file",
+			"/C:/folder/file": "C:\\folder\\file",
+			"/folder/file":    "C:\\folder\\file",
+			"file":            "C:\\file",
+			"/file":           "C:\\file",
+			"D:/folder/file":  "",
+		},
+		"/C:": {
```

Comment:
Nope, old `vfs.OS` doesn't work with `/C:`. With the old implementation:

```
got `\C:\foo` while resolving `foo` on `/C:`
got `\C:\foo` while resolving `/foo` on `/C:`
```
which cannot be translated to real OS path when trying to access file content